frontend/src/metabase/status/components/StatusLarge/StatusLarge.tsx

```diff
@@ -0,0 +1,109 @@
+import React from "react";
+import Ellipsified from "metabase/core/components/Ellipsified";
+import Icon from "../../../components/Icon";
+import useStatusVisibility from "../../hooks/use-status-visibility";
+import {
+  StatusCardRoot,
+  StatusCardIcon,
+  StatusCardBody,
+  StatusCardTitle,
+  StatusCardDescription,
+  StatusCardSpinner,
+  StatusCardIconContainer,
+  StatusRoot,
+  StatusHeader,
+  StatusTitle,
+  StatusToggle,
+  StatusBody,
+} from "./StatusLarge.styled";
+
+type status = {
+  title: string;
+  items: statusItem[];
+};
+
+type statusItem = {
```

Comment:
Pascal case for types is used in other places, wdyt about sticking to it? Also, these could be interfaces